README.md

```diff
@@ -0,0 +1,22 @@
+# Deep Dungeon Compendium
+
+Contributors please read [contributing guidelines.](./contributing.md)
+
+## Local Development
+
+Pre-requisites:
+
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+- [Jekyll & Dependencies](https://jekyllrb.com/docs/installation/)
+
+To run:
+
+```sh
+  bundle exec jekyll serve
+```
+
+To build search index:
+
+```sh
+  ./scripts/build-search-data.bash
+```
```

Comment:
Please remove this bit to avoid confusion as it doesn't exist yet. We can add it back later. Thanks!